libc/printf/stdio_printf.c

```diff
@@ -2036,7 +2036,9 @@ TEST(stdio_printf_cspn, S)
 
 TEST(stdio_printf_cspn, p)
 {
-#ifdef __phoenix__
+#if defined(__phoenix__) && defined(__CPU_ZYNQMP)
```

Comment:
It should be done in a separate PR - if it's the temporary change needed to see the results from CI - it should be placed in the temporary commit within this PR (but then you should just check the results and after that move it to the valid PR).

Reply:
don't depend on `__CPU`, this should be true for all 64-bit architectures.